models/spring-ai-deepseek/src/main/java/org/springframework/ai/deepseek/api/DeepSeekApi.java

```diff
@@ -484,12 +481,13 @@ public void setJsonSchema(String jsonSchema) {
 	 * @param toolChoice Controls which (if any) function is called by the model. none
 	 * means the model will not call a function and instead generates a message. auto
 	 * means the model can pick between generating a message or calling a function.
-	 * Specifying a particular function via {"type: "function", "function": {"name":
+	 * Specifying a particular function via {"type": "function", "function": {"name":
 	 * "my_function"}} forces the model to call that function. none is the default when no
 	 * functions are present. auto is the default if functions are present. Use the
 	 * {@link ToolChoiceBuilder} to create the tool choice value.
 	 */
 	@JsonInclude(Include.NON_NULL)
+	@JsonIgnoreProperties(ignoreUnknown = true)
```

Comment:
We want to add this annotation only for the response objects. Hence, we can remove this. Will fix it when merging.